pkg/sources/git/git.go

```diff
@@ -80,8 +80,11 @@ func NewGit(sourceType sourcespb.SourceType, jobID sources.JobID, sourceID sourc
 }
 
 // Ensure the Source satisfies the interfaces at compile time.
-var _ sources.Source = (*Source)(nil)
-var _ sources.SourceUnitUnmarshaller = (*Source)(nil)
+var _ interface {
+	sources.Source
+	sources.SourceUnitEnumChunker
+	sources.SourceUnitUnmarshaller
+} = (*Source)(nil)
```

Comment:
whoa i didn't know you could smoosh interfaces together like this

Reply:
It's an anonymous interface composed of other interfaces! Similar to `interface{}` (aka `any`) but with stuff in it.